### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YuliiaSaliuta**
CarLs
CarLsAt
DL
ML
MLM

### CODE REVIEW

**Review by Kateryna-Bordonos17 — CHANGES_REQUESTED** [review]
you need to add query transform and edit the pagination so that the search results are saved when you switching pages

File: tests/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_user(
+            username="driver",
+            password="driver1234",
+            license_number="ABC12345"
+        )
+
+    def test_driver_license_number_listed(self):
+        url = reverse("admin:taxi_driver_changelist")
+        res = self.client.get(url)
```
Comment (LiudmylaKulinchenko) [inline tests/test_admin.py:22]: use descriptive names, not res

File: tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    def test_manufacturer_meta_ordering(self):
+        expected_ordering = ["name"]
+        manufacturer_ordering = Manufacturer._meta.ordering
+        self.assertListEqual(manufacturer_ordering, expected_ordering)
+
+    def test_driver_str(self):
+        driver = Driver.objects.get(id=1)
+        expected_object_name = f"{driver.username} " \
+                               f"({driver.first_name} {driver.last_name})"
```
Comment (LiudmylaKulinchenko) [inline tests/test_models.py:38]: Don't use `\` as a line breaker, use parentheses instead

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    def test_driver_login_required(self):
+        res = self.client.get(DRIVER_URL)
+
+        self.assertNotEqual(res.status_code, 200)
+
+    def test_car_login_required(self):
+        res = self.client.get(CAR_URL)
+
+        self.assertNotEqual(res.status_code, 200)
+
```
Comment (LiudmylaKulinchenko) [inline tests/test_views.py:26]: Don't use res

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by Daniil-Pankieiev — APPROVED** [review]

**Review by prochigor — APPROVED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job 🚀

**Pull request closed without merging**